rasa_core/remote.py

```diff
@@ -125,6 +125,28 @@ def parse(self, message, sender_id):
                                               response.text))
             return None
 
+    def respond(self, sender_id, message):
+        # type: (Text, Text) -> Optional[Dict[Text, Any]]
+        """Send a parse request to a rasa core server."""
+
+        url = "/conversations/{}/respond".format(sender_id)
```

Comment:
seems to be a lot of duplicating code with `parse(..)` except for this line, should it be one function with request type parameter?

Reply:
and `continue_core` seems to be the same except for `data`